internal/grpcutil/target.go

```diff
@@ -41,29 +41,52 @@ func split2(s, sep string) (string, string, bool) {
 // not parse "unix:[path]" cases. This should be true in cases where a custom
 // dialer is present, to prevent a behavior change.
 //
-// If target is not a valid scheme://authority/endpoint, it returns {Endpoint:
-// target}.
+// If target is not valid dns/unix/unix-abstract schemas as specified in
+// https://github.com/grpc/grpc/blob/master/doc/naming.md,
```

Comment:
This isn't quite accurate.  We support custom schemes, and those will parse into scheme/authority/endpoint as well.  I think you can leave the old comment how it was, just add the reference link.

Reply:
done